### DISCUSSION TITLE
To-Do list

### DISCUSSION THREAD

**Discussion by ykawashima** [HajimeKawahara/exojax#258] [Ideas]
## spec/modit.py
- Prepare the DIT Grid Matrix of HITRAN/HITEMP also for the self-pressure axis in setdgm_hitran?

## spec/moldb.py
- Add a warning in Qr_line_HAPI_jax when the input temperature is outside the temperature range of the original HAPI partition function data

**Comment by HajimeKawahara** [comment]
## spec/premodit.py
- include natural widths　#260
